java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -74,6 +76,45 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> getdel(String key);
 
+    /**
+     * Gets the value associated with the given <code>key</code>. <code>GETEX</code>is similar to
+     * {@see #get}.
```

Comment:
Personally I think we can omit the part mentioning its similarity to get, same goes for the other docs
```suggestion
     * Gets the value associated with the given <code>key</code>.
```

Reply:
if keeping that part, fix spacing
```suggestion
     * Gets the value associated with the given <code>key</code>. <code>GETEX</code> is similar to
     * {@see #get}.
```